### ISSUE TITLE
Does this code runs on Python 3

### ISSUE THREAD

**Issue by arqeco** [mafm/ledger.py#23]
Hello,

Is this project alive? Does this code runs in Python 3?

**Comment by lslavkov** [comment]
I don't think that this project is alive. Last commit was at June 2018. It would be the best to use original [ledger-cli](https://ledger-cli.org/)

**Comment by finsteininvest** [comment]
I did a conversion: https://github.com/finsteininvest/ledger3
